ccd-definition/AuthorisationCaseField/systemUpdate.json

```diff
@@ -149,6 +149,12 @@
     "UserRole": "caseworker-civil-systemupdate",
     "CRUD": "R"
   },
+  {
```

Comment:
Should this not be part of systemUpdate-nonprod.json?

Reply:
old comment but non-prod files not needed